safe-api-fetch/src/index.js

```diff
@@ -2,6 +2,7 @@
 import * as SafeAPI from "safe-api";
 import queryString from "querystring";
 import fetch from "isomorphic-fetch";
+import _ from "lodash";
```

Comment:
use vanilla for-loops (better typing, smaller bundle size, fewer ways to do things, fewer APIs to know to to read to codebase)

Reply:
🔨 :milk_glass: 🔥